### PR TITLE
[INLONG-10391][Manager] Supports configuring zk clusters and issuing zk addresses to agents

### LINKED ISSUE #10391: [Improve][Manager] Supports configuring zk clusters and issuing zk addresses to agents
### Description

Supports configuring zk clusters and issuing zk addresses to agents.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
Fixes #10391

### Motivation

 Supports configuring zk clusters and issuing zk addresses to agents.

### Modifications

 Supports configuring zk clusters and issuing zk addresses to agents.

### CODE REVIEW

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/zk/AgentZkClusterDTO.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+@NoArgsConstructor
+@ApiModel("Agent zk cluster info")
+public class AgentZkClusterDTO {
+
+    /**
+     * Get the dto instance from the request
+     */
+    public static AgentZkClusterDTO getFromRequest(AgentZkClusterRequest request) {
+        return AgentZkClusterDTO.builder()
+                .build();
```
Comment (aloyszhang) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/zk/AgentZkClusterDTO.java:45]: Plz keep the logic consistent with the descriptions of docs `Get the dto instance from the request`.

Comment (fuweng11) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/zk/AgentZkClusterDTO.java]: Fixed.

File: inlong-common/src/main/java/org/apache/inlong/common/pojo/agent/AgentConfigRequest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class AgentConfigRequest {
+
+    private String clusterTag;
+
+    private String clusterName;
+
+    private String ip;
```
Comment (aloyszhang) [inline inlong-common/src/main/java/org/apache/inlong/common/pojo/agent/AgentConfigRequest.java:36]: what this field for?

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java [hunk 3]
```diff
@@ -322,6 +328,34 @@ private void updateTaskStatus(CommandEntity command) {
         }
     }
 
+    @Override
+    public AgentConfigInfo getAgentConfig(AgentConfigRequest request) {
+        LOGGER.info("begin to get init info for: " + request);
```
Comment (aloyszhang) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java:333]:
```suggestion
        LOGGER.info("begin to get init info for {}", request);
```

Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java]: Fixed.

File: inlong-common/src/main/java/org/apache/inlong/common/pojo/agent/AgentConfigRequest.java (hunk 2 above)
Comment (fuweng11) [reply inlong-common/src/main/java/org/apache/inlong/common/pojo/agent/AgentConfigRequest.java]: Fixed.

**Review by aloyszhang — DISMISSED** [review]

**Review by dockerzhang — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/AgentZkClusterOperator.java [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+            CommonBeanUtils.copyProperties(dto, info);
+        }
+        return info;
+    }
+
+    @Override
+    public Object getClusterInfo(InlongClusterEntity entity) {
+        AgentZkClusterInfo agentZkClusterInfo = (AgentZkClusterInfo) this.getFromEntity(entity);
+        Map<String, String> map = new HashMap<>();
+        map.put("url", agentZkClusterInfo.getUrl());
```
Comment (justinwwhuang) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/AgentZkClusterOperator.java:91]: url should be defined using macros?

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java [hunk 5]
```diff
@@ -322,6 +328,34 @@ private void updateTaskStatus(CommandEntity command) {
         }
     }
 
+    @Override
+    public AgentConfigInfo getAgentConfig(AgentConfigRequest request) {
+        LOGGER.info("begin to get agent config info for {}", request);
```
Comment (justinwwhuang) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java:333]: Will there be too many logs printed like this?

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/AgentZkClusterOperator.java (hunk 4 above)
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/AgentZkClusterOperator.java:91]: Here is a return map that supports extensions.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java (hunk 5 above)
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java]: Fixed.

**Review by justinwwhuang — APPROVED** [review]

**Review by aloyszhang — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**